cpp/oneapi/dal/algo/triangle_counting/backend/cpu/vertex_ranking_default_kernel_scalar.hpp

```diff
@@ -107,68 +99,71 @@ ONEDAL_FORCEINLINE array<std::int64_t> triangle_counting_local_(
                     }
                     if (w == *u_neighbors_ptr) {
                         int thread_id = dal::detail::threader_get_current_thread_index();
-                        int64_t indx = (int64_t)thread_id * (int64_t)g_vertex_count;
-                        triangles_local[indx + u]++;
-                        triangles_local[indx + v]++;
-                        triangles_local[indx + w]++;
+                        int64_t index_stride = (int64_t)thread_id * (int64_t)vertex_count;
+                        triangles_local[index_stride + u]++;
+                        triangles_local[index_stride + v]++;
+                        triangles_local[index_stride + w]++;
                     }
                 }
             }
         });
     }
     else { //average_degree >= average_degree_sparsity_boundary
-        dal::detail::threader_for_simple(g_vertex_count, g_vertex_count, [&](std::int32_t u) {
-            if (g_degrees[u] >= 2)
+        dal::detail::threader_for_simple(vertex_count, vertex_count, [&](std::int32_t u) {
+            if (get_topology_vertex_degree(t, u) >= 2)
                 dal::detail::threader_for_int32ptr(
-                    g_vertex_neighbors + g_edge_offsets[u],
-                    g_vertex_neighbors + g_edge_offsets[u + 1],
+                    get_topology_vertex_neighbors_begin(t, u),
```

Comment:
the only minor concern - too long name of the function

Reply:
Yes. Name can be shorter. We can remove topology and to have the same name of service as the public graph service beside used namespace detail.